docs/dataset/quickstart.md

```diff
@@ -5,3 +5,185 @@ icon: kolena/flame-16
+In this quickstart guide, we'll use we will use the [300 Faces In the Wild (300-W)](https://ibug.doc.ic.ac.uk/resources/300-W/)
+example dataset to demonstrate the how to curate test data and test models in Kolena. The examples
+use data located in a public S3 bucket `s3://kolena-public-examples`
+
+## Installing `kolena` Python SDK
+
+=== "`pip`"
+
+    ```shell
+    pip install kolena
+    ```
+
+=== "`poetry`"
+
+    ```shell
+    poetry add kolena
+    ```
+
+## Clone the examples
+
+The [kolenaIO/kolena](https://github.com/kolenaIO/kolena) repository contains a number of example integrations to clone
+and run directly:
+
+<div class="grid cards" markdown>
+
+- [:kolena-keypoint-detection-20: Example: Keypoint Detection ↗](https://github.com/kolenaIO/kolena/tree/trunk/examples/dataset/keypoint_detection)
+
+    ![Example image and five-point facial keypoints array from the 300 Faces in the Wild dataset.](../assets/images/300-W.jpg)
+
+    ---
+
+    End-to-end face recognition 1:1 using the [Labeled Faces in the Wild (LFW)](http://vis-www.cs.umass.edu/lfw/) dataset.
+- [:kolena-chat-20: Example: Question Answering ↗](https://github.com/kolenaIO/kolena/tree/trunk/examples/dataset/question_answering)
+
+    ![Example questions from CoQA dataset.](../assets/images/CoQA.jpg)
+
+    ---
+
+    Question Answering using the
+    [Conversational Question Answering (CoQA)](https://stanfordnlp.github.io/coqa/) dataset
+</div>
+
+Now, let's set up the keypoint detection example:
+
+```shell
+cd kolena/examples/object_detection_2d
+poetry update && poetry install
+```
+
+## Dataset Upload
+
+Model evaluations on Kolena starts with datasets. Datasets are tables that contain the data you wish to use for
+creating test cases.
+
+=== "SDK"
+
+    The example code contains a script `keypoint_detection/upload_dataset.py` which will process the CSV
+    file `s3://kolena-public-examples/300-`
+    and register it as a dataset in Kolena using the `register_dataset` function.
+
+    First, let's first configure our environment by populating the `KOLENA_TOKEN`
+    environment variable. Visit the [:kolena-developer-16: Developer](https://app.kolena.io/redirect/developer) page to
+    generate an API token and copy and paste the code snippet into your environment:
+
+    ```shell
+    export KOLENA_TOKEN="********"
+    ```
+
+    We can now register a new dataset using the provided script:
+
+    ```shell
+    poetry run python3 keypoint_detection/upload_dataset.py
+    ```
+
+    After this script has completed, a new dataset named "300-W" will be created, which you can
+    see in [:kolena-dataset-20: Datasets](https://app.kolena.io/redirect/datasets)
+
+=== "Web App"
+
+    To import the 300-W dataset, select “Import datasets” then “Select from cloud storage”. Using the explorer, navigate to
+    s3://kolena-pubic-examples/300-W/ and select 300-W.csv.
+
+    Note: The keypoints in `300-W.csv` have been defined using `kolena.workflow.annotation.Keypoints` from
```

Comment:
Does it mean that the example script should be using a different file, i.e. in the raw folder?

Reply:
Yes, that's my understanding of this thread https://kolena-io.slack.com/archives/C01GAKM1W5S/p1704394991235389